### ISSUE TITLE
TOC scroll chasing failed on the homepage if 'disqus' is set.

### ISSUE THREAD

**Issue by jfcherng** [squidfunk/mkdocs-material#730]
## Description

My `disqus` is set in the yaml config like
```yaml
extra:
  disqus: 'my_disqus_id'
```

The homepage is hard-coded to not show the Disqus forum.

https://github.com/squidfunk/mkdocs-material/blob/0d788714c6f169e412b8d18646444f7b3980392a/src/base.html#L298-L301

However, in the homepage, if `disqus` is set, there would be a `Comment` section in TOC.

After scrolling down and up in the homepage, my Chrome console shows

```
application.8eb9be28.js:1 Uncaught TypeError: Cannot read property 'offsetTop' of undefined
    at e.update (application.8eb9be28.js:1)
    at update_ (application.8eb9be28.js:1)
```

and then the TOC scroll chasing failed.

### Expected behavior

The `Comment` section should not be shown in homepage's TOC.

### Actual behavior

It shows.

### Steps to reproduce the bug

1. Set any disqus id in the yaml config
2. Build the site

### Package versions

* Python: Python 3.6.4
* MkDocs: mkdocs, version 0.17.2
* Material: Version: 2.7.0

**Comment by squidfunk** [comment]
Thanks, that's indeed a bug.

**Comment by squidfunk** [comment]
Fixed in 6ecf71ee8d830880afc1d12572f005047173dbd7

**Comment by squidfunk** [comment]
Fix released as part of 2.7.1
